client/my-sites/marketplace/components/billing-interval-switcher/constants.tsx

```diff
@@ -0,0 +1,4 @@
+export enum IntervalLength {
+	ANNUALLY = 'ANNUALLY',
+	MONTHLY = 'MONTHLY',
+}
```

Comment:
Thanks for address this, can we change those values to `lowercase` so we can use them directly when looking on plugin variations?